src/panel/direction/RouteSummary.jsx

```diff
@@ -40,31 +40,32 @@ export default class RouteSummary extends React.Component {
     const { route, vehicle } = this.props;
 
     return <div className="itinerary_leg_summary" onClick={this.onClick}>
-      <div className={`itinerary_leg_icon ${getVehicleIcon(vehicle)}`} />
-      <div className="itinerary_leg_via">
-        <RouteVia route={route} vehicle={vehicle} />
+
+      <Flex>
+        <div className={`itinerary_leg_icon ${getVehicleIcon(vehicle)}`} />
+        <RouteSummaryInfo
+          route={route}
+          vehicle={vehicle}
+        />
+      </Flex>
```

Comment:
Looks like all elements could be in a single Flex, but as there are other redesign steps coming, let's wait a bit.